cmd/src/search.go

```diff
@@ -549,12 +598,13 @@ const searchResultsTemplate = `{{- /* ignore this line for template formatting s
 
 			{{- /* Line matches */ -}}
 			{{- $lineMatches := .lineMatches -}}
+			{{- $content := .file.content -}}
```

Comment:
![](https://media3.giphy.com/media/SRx5tBBrTQOBi/giphy.gif?cid=5a38a5a22f16080aa60021a8b6d9658c2b16c09470883730&rid=giphy.gif)

I don't know how this template language works and did not care to look deeply into it, I just did things based on other stuff I saw